README.rst

```diff
@@ -88,11 +88,22 @@ Installation
 
        easy_install aiosparql
 
+
 Requirements
 ^^^^^^^^^^^^
 
  *  Python >= 3.5
 
+Testing
+-------
+
+In order for the tests to run, you should have the following docker container started:
+
+   ::
+   
+      docker run -d --name travis-virtuoso -p 8890:8890 -e SPARQL_UPDATE=true tenforce/virtuoso:1.2.0-virtuoso7.2.2
```

Comment:
Actually you now need both containers to be started I think. Right?

Reply:
Thanks, didn't get back to add that yet